tests/lapi/lib.lua

```diff
@@ -26,6 +26,22 @@ local function version()
     return t[1] or t[1/0] or t[f()==f()]
 end
 
+local function MAX_INT()
+    local max_integer = 2^51
+    if version() ~= "LuaJIT" then
+        max_integer = math.maxinteger
+	end
+	return max_integer
+end
+
+local function MIN_INT()
+    local min_integer = -2^51
+    if version() ~= "LuaJIT" then
```

Comment:
It should be done with `_VERSION` check.
`math.maxinteger`, `math.mininteger` are introduced in Lua 5.3.

Reply:
> It should be done with `_VERSION` check.

LuaJIT runtimes reports version "Lua 5.1", but it is obviously not so:

```
t[0] ~ $ tarantool
Tarantool 2.11.5-0-g12a9ceb870c
type 'help' for interactive help
tarantool> _VERSION
---
- Lua 5.1
...

tarantool> 

[0] ~ $ luajit 
LuaJIT 2.1.1703358377 -- Copyright (C) 2005-2023 Mike Pall. https://luajit.org/
JIT: ON SSE3 SSE4.1 BMI2 fold cse dce fwd dse narrow loop abc sink fuse
> print(_VERSION)
Lua 5.1
> 
```

Reply:
Yes, but LuaJIT is compatible with Lua 5.1 API, not 5.3. See no contradiction here.